src/App.tsx

```diff
@@ -115,6 +109,7 @@ const App: React.FC = () => {
         }
       })
       .catch((e) => {
+        console.error(e);
```

Comment:
remove

Reply:
it's useful for dev... but I can wrap in a flag